src/wallet/wallet_errors.h

```diff
@@ -1001,3 +1003,33 @@ namespace tools
     LOG_ERROR(#cond << ". THROW EXCEPTION: " << #err_type);                                                 \
     tools::error::throw_wallet_ex<err_type>(std::string(__FILE__ ":" STRINGIZE(__LINE__)), ## __VA_ARGS__); \
   }
+
+namespace tools
+{
+  namespace error
+  {
+    static inline void throw_on_rpc_response_error(bool r, const epee::json_rpc::error &error, const std::string &status, const char *method)
```

Comment:
Again, I would drop the `static` keyword here. This one should arguably be in a cpp anyway.